src/classes/BDI_DataImportService.cls

```diff
@@ -962,7 +962,7 @@ global with sharing class BDI_DataImportService {
 
         // now update the Contacts to create their Affiliations
         if (mapConIdToConUpdate.size() > 0) {
-            update mapConIdToConUpdate.values();
+            UTIL_DMLService.updateRecords(mapConIdToConUpdate.values());
```

Comment:
This class has lots of other places where we insert/update/upsert Accounts, Addresses, Campaigns, and CampaignMembers.  do you think some or all of those should also be going thru UTIL_DMLService to respect the new setting?  I'm guessing Accounts is the likely one to be problematic (ie, the user may have specified a rule).  But maybe worth doing for all for consistency?  what are your thoughts?  thanks.

Reply:
Added the code for consistency. Good call.